src/Widgets/ComponentRow.vala

```diff
@@ -80,48 +80,54 @@ public class DateTime.ComponentRow : Gtk.ListBoxRow {
             }
         }
 
-        unowned string icon_name = "office-calendar-symbolic";
+        var icon_name = "office-calendar-symbolic";
         if (source_selectable is E.SourceTaskList) {
             icon_name = "office-task-symbolic";
         } else if (end_time == null) {
             icon_name = "alarm-symbolic";
         }
 
-        component_image = new Gtk.Image.from_icon_name (icon_name, Gtk.IconSize.MENU);
-        component_image.valign = Gtk.Align.START;
+        component_image = new Gtk.Image.from_icon_name (icon_name, Gtk.IconSize.MENU) {
+            valign = Gtk.Align.START
+        };
 
-        unowned Gtk.StyleContext component_image_context = component_image.get_style_context ();
+        var component_image_context = component_image.get_style_context ();
```

Comment:
Strictly this should be `unowned var` although it makes little difference in this case (just temporarily increases the reference count on the returned stylecontext) and you can find examples of both usages in the Elementary codebase.